build.gradle

```diff
@@ -115,27 +117,17 @@ project(":plugin.idea") {
 
     jar {
         baseName 'com.microsoft.alm.plugin.idea'
+        // For build plugin without developer telemetry -Dorg.gradle.project.developer_telemetry=false
+        if (developer_telemetry != "true") {
+            exclude 'telemetry/'
+        }
         from('.') {
             include 'META-INF/plugin.xml'
         }
     }
 
-    task delTestTelemetryPropFile << {
-        delete 'resources/telemetry/com.microsoft.alm.plugin-telemetry.properties'
-    }
-
-    /**
-     * Create the plugin zip
-     */
-    task zip(type: Zip, dependsOn: ['test', 'check']) {
```

Comment:
Can we please keep the zip task?  We use it on our side.

Reply:
Yes, I can but IDEA SDK has task buildPlugin for this purpose. Maybe could you use it instead of zip task?

Reply:
Thanks for the tip!  It would be great if we can leave the zip task here for now so we can easily validate the change.  We can look update the build on our side at a later date.

Reply:
Sure, no problem.